parser/js/test/benchmark/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    displayName: 'Benchmarking',
+    transform: {
+        "^.+\\.ts": "ts-jest",
+    },
+    testRegex: "(/test/benchmarking/.*|(\\.|/)(test|spec))\\.(js|ts)$",
+    moduleFileExtensions: ["ts", "js", "json", "node"],
+};
```

Comment:
If I'm understanding right, I think a file like this is not needed as the benchmark scripts are not jest tests.

Reply:
Yes, good catch!